pkg/installer/generateconfig.go

```diff
@@ -148,9 +158,14 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 						Zones:            masterZones,
 						InstanceType:     string(m.oc.Properties.MasterProfile.VMSize),
 						EncryptionAtHost: m.oc.Properties.MasterProfile.EncryptionAtHost == api.EncryptionAtHostEnabled,
+						VMNetworkingType: "Basic",
```

Comment:
Is there a reason we are forcing this to basic?

Reply:
because there were concerns from team members about us implementing accelerated networking in the past that led to outages - it was suggested that we put any accelerated networking capabilities behind a toggle-able flag we could revert at any time, which would require an API release.